frontends/web/src/components/devices/bitbox02/settings.tsx

```diff
@@ -34,11 +34,12 @@ interface SettingsProps {
 
 interface State {
     versionInfo?: VersionInfo;
-    deviceInfo: {
+    deviceInfo?: {
```

Comment:
Hm... so how did it work before this, without the `?`?

Reply:
Yeah there is even a `!== undefined` check somewhere, which I would expect the linter to trigger if it cannot be `undefined`. :shrug: